### PR TITLE
Dashboard fixed

### PR DESCRIPTION
**Pull request by jimccfun**
**What this PR does / why we need it**:
1. Identity module tenant permission modification;
2. Profile module params format
3. Modify some page style

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
@xing-yang @leonwanghui 

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by xing-yang** [comment]
Have you done a rebase?  Why does it show so many older commits?

File: dashboard/src/app/business/home/home.component.ts [hunk 1]
```diff
@@ -283,5 +292,6 @@ export class HomeComponent implements OnInit {
         let tenantId = this.paramStor.CURRENT_TENANT().split("|")[1];
         this.getAllvolumes(tenantId);
         this.getAllSnapshots(tenantId);
+        this.getAllReplication(tenantId);
```
Comment (xing-yang) [inline dashboard/src/app/business/home/home.component.ts]: getAllReplications

File: dashboard/src/app/business/profile/createProfile/createProfile.component.ts [hunk 2]
```diff
@@ -369,9 +367,9 @@ export class CreateProfileComponent implements OnInit {
         this.msgs = [];
         this.msgs.push({ severity: 'info', summary: 'Success', detail: 'Form Submitted' });
         this.param.name = value.name;
-        this.param.storageType = value.storageType;
-        this.param.extras.protocol = value.protocol;
-        this.param.extras.policys = value.policys;
+        // this.param.storageType = value.storageType;
+        // this.param.extras.protocol = value.protocol;
+        // this.param.extras.policys = value.policys;
```
Comment (xing-yang) [inline dashboard/src/app/business/profile/createProfile/createProfile.component.ts]: Why are these commented out?

**Comment by xing-yang** [comment]
I copied the "dashboard" folder from your development branch to my setup and deployed it.  After that, the profile page does not work at all.

**Comment by wisererik** [comment]
@leonwanghui  you can use `squash and merge` command to squash all commit into one commit.

File: dashboard/src/app/business/profile/createProfile/createProfile.component.ts (hunk 2 above)
Comment (wisererik) [reply dashboard/src/app/business/profile/createProfile/createProfile.component.ts]: I think these capabilities was moved into ioConnectivityLoS and dataStorageLoS

**Comment by jimccfun** [comment]
@xing-yang @leonwanghui    PR updated, please handle it.

**Pull request merged**